chapters/data-collection.tex

```diff
@@ -1,362 +1,251 @@
+
+Once the content of the questionnaire is finalized and translated, it is time to proceed with programming the electronic survey instrument.
 
 
 %------------------------------------------------
+\section{Programming CAPI questionnaires}
+Electronic data collection has great potential to simplify survey implementation and improve data quality.
+Electronic questionnaires are typically created in a spreadsheet (e.g. Excel or Google Sheets) or software-specific form builder, accessible even to novice users. \sidenote{\url{https://dimewiki.worldbank.org/wiki/Questionnaire_Programming}}
+We will not address software-specific form design in this book; rather, we focus on coding conventions that are important to follow for CAPI regardless of software choice.
+\sidenote{\url{https://dimewiki.worldbank.org/wiki/SurveyCTO_Coding_Practices}}
+
+CAPI software tools provide a wide range of features designed to make implementing even highly complex surveys easy, scalable, and secure.
+However, these are not fully automatic: you still need to actively design and manage the survey.
+Here, we discuss specific practices that you need to follow to take advantage of CAPI features and ensure that the exported data is compatible with the software that will be used for analysis, and the importance of a data-focused pilot.
+
+\subsection{CAPI workflow}
+The starting point for questionnaire programming is a complete paper version of the questionnaire, piloted for content, and translated where needed.
+Starting the programming at this point reduces version control issues that arise from making significant changes to concurrent paper and electronic survey instruments.
+When you start programming, do not start with the first question and program your way to the last question.
+Instead, code from high level to small detail, following the same questionnaire outline established at design phase.
+The outline provides the basis for pseudocode, allowing you to start with high level structure and work down to the level of individual questions. This will save time and reduce errors.
+
+\subsection{CAPI features}
+CAPI surveys are more than simply an electronic version of a paper questionnaire.
+All common CAPI software allow you to automate survey logic and add in hard and soft constraints on survey responses.
+These features make enumerators' work easier, and they create the opportunity to identify and resolve data issues in real-time, simplifying data cleaning and improving response quality.
+Well-programmed questionnaires should include most or all of the following features:
+
+\begin{itemize}
+	\item{\textbf{Survey logic}}: build all skip patterns into the survey instrument, to ensure that only relevant questions are asked. This covers simple skip codes, as well as more complex interdependencies (e.g., a child health module is only asked to households that report the presence of a child under 5)
```

Comment:
"_build all skip patterns into the survey instrument_" in opposed to what? Relying on human? I think that can be made explicit